src/scripts/clipperUI/frontEndGlobals.ts

```diff
@@ -16,13 +16,31 @@ export module Clipper {
 	}
 
 	export module Storage {
-		export function getValue(key: string, callback: (value: string) => void): void {
+		let storageCache: { [key: string]: string } = {};
+
+		export function preCacheValues(storageKeys: string[]): void {
+			for (let key of storageKeys) {
+				Clipper.Storage.getValue(key, () => { }, true);
```

Comment:
I know Matthew made a `getValues` but you'd have to add in the caching in again that you already added to `getValue` (if we want to switch to that function). Just a heads up depending on who merges first.